urllib3/util/ssl_.py

```diff
@@ -17,6 +17,23 @@
 
 import errno
 import warnings
+import hmac
+
+if hasattr(hmac, 'compare_digest'):
```

Comment:
How about something like:

```python
def _const_compare_digest_backport(a, b):
    ...

_const_compare_digest = getattr(hmac, 'compare_digest', _const_compare_digest_backport)
```

This way we can have a test that exercises `_const_compare_digest_backport`?